app/models/author.rb

```diff
@@ -0,0 +1,11 @@
+class Author < ApplicationRecord
```

Comment:
By mentioning `ActiveModel::Validations` I hinted that the form is model-less and Author is not a descendant of `ApplicationRecord`. Why would we create Author model as the descendant of ApplicationRecord if we don't need to store input in DB? And you don't need to include `ActiveModel::Validations` in descendant of ApplicationRecord explicitly.

Reply:
That part surely had been confusing me. Thanks for making it clear. I will try my best to make it correct.